jablib/src/main/java/org/jabref/logic/util/io/RegExpBasedFileFinder.java

```diff
@@ -166,12 +166,9 @@ private List<Path> findFile(final BibEntry entry, final Path directory, final St
             }
             switch (dirToProcess) {
                 case "." -> {
-                    continue;  // Stay in current directory
-                }
-                case ".." -> {
-                    currentDirectory = currentDirectory.getParent();
-                    continue;
+                    // Stay in current directory
                 }
+                case ".." -> currentDirectory = currentDirectory.getParent();
                 case "*" -> { // for all direct subdirs
```

Comment:
May be functionally correct, but I think it was more easily understandable before.
Now you have three cases with a statement block and one in the middle without.
Before you know, with continue, do nothing else, next dir, now you look for the end of the swith an look, if there is any code.